conda/core/solve.py

```diff
@@ -8,12 +8,11 @@
 import sys
 from itertools import chain
 from logging import DEBUG, getLogger
-from os.path import join
+from os.path import exists, join
 from textwrap import dedent
 from typing import TYPE_CHECKING
 
 from boltons.setutils import IndexedSet
-from genericpath import exists
```

Comment:
`os.path` is actually either `posixpath` or `ntpath` based on the OS we are on and both of import generic functions (like `exists`) from `genericpath`